bragg-edge-imaging-2D.ipynb

```diff
@@ -124,15 +124,12 @@
     "\n",
     "import ess.v20.imaging as imaging\n",
     "import ess.v20.imaging.operations as operations\n",
-    "import scippconfig\n",
+    "import dataconfig\n",
     "from scipy import ndimage, signal\n",
     "\n",
-    "DATA_DIR_NAME = \"stress-experiments\"\n",
-    "OUTPUT_DIR_NAME = \"output\"\n",
-    "\n",
-    "data_dir = os.path.join(scippconfig.script_root, DATA_DIR_NAME)\n",
-    "\n",
-    "output_dir = os.path.join(scippconfig.script_root, OUTPUT_DIR_NAME)\n",
+    "local_data_path = os.path.join('ess', 'v20', 'imaging', 'stress-experiments')\n",
```

Comment:
I think python3 is shifting towards using the [pathlib](https://docs.python.org/3/library/pathlib.html) library rather than os.path module. We may want to think about whether to emulate this shift but for basic path manipulations like this it makes little odds.

Reply:
Indeed. I think it's not worthwhile to upgrade at this point.